content/landing/done-for-you-demo/_how.md

```diff
@@ -1,10 +1,10 @@
 ## Here's How It Works
 
-You let us know what comes up again and again in support requests and point us towards relevant documentation.
+You let us know what comes up in support requests and point us towards relevant documentation.
 
 We propose a feature set for the demo showcasing solutions to the most costly support request.
 
-You approve the feature set and decide on the demo framework (i.e., Gatsby, Next, or plain React).
+You approve the feature set and decide to get this Gatsby demo started.
```

Comment:
Maybe:

You approve the feature set and **we** the Gatsby demo started.

Reply:
You approve the feature set and we get the Gatsby demo started.